src/main/java/com/stripe/model/Transfer.java

```diff
@@ -303,4 +303,21 @@ public Transfer update(Map<String, Object> params, RequestOptions options)
         Transfer.class, options);
   }
   // </editor-fold>
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Summary extends StripeObject {
```

Comment:
Similar as the other one, it's likely time to remove support for this one entirely too. This is unrelated to this PR for sure but it's the best opportunity I found to flag some of those

Reply:
It does mean the library would become partially incompatible with those very old API versions though.

Reply:
IMO it already is in the sense that there are so many old cases we have no idea exist and might have broken or never supported though.
I'm fine leaving as it, but I do worry that we don't have any tests for things like this and end up thinking we support it when we might not be doing it properly